docs/sections/user_guide/yaml/components/make_solo_mosaic.rst

```diff
@@ -0,0 +1,41 @@
+.. _make_solo_mosaic_yaml:
+
+make_solo_mosaic
+================
+
+Structured YAML to run the WRF preprocessing component ``make_solo_mosaic`` is validated by JSON Schema and requires the ``make_solo_mosaic:`` block, described below. If ``make_solo_mosaic`` is to be run via a batch system, the ``platform:`` block, described :ref:`here <platform_yaml>`, is also required.
+
+Documentation for the UFS Utils ``make_solo_mosaic`` program is :ufs-utils:`here <make-solo-mosaic>`.
+
+Here is a prototype UW YAML ``make_solo_mosaic:`` block, explained in detail below:
+
+.. highlight:: yaml
+.. literalinclude:: ../../../../shared/make_solo_mosaic.yaml
+
+UW YAML for the ``make_solo_mosaic:`` Block
+-------------------------------------------
+
+config:
+^^^^^^^
+
+Describes the required parameters to run a ``make_solo_mosaic`` configuration.
```

Comment:
I see in the prototype YAML config below that there are several parameters below `config:`. Should we list and describe them here, or at least list them and give a link to external docs describing what each means?